api/src/lib.rs

```diff
@@ -1,7 +1,11 @@
+mod dto;
```

Comment:
What is the dto acronym?

Reply:
It's data transfer object.. a coworker at my last job used it a lot (c# background). I was going to call it `types` even though it's a bad name... I basically wanted to signal that these are public types that are meant to be sent over the wire, but it didn't even really turn out like that since ResumeToken isn't in a public API interface (just built from a string body). 

So anyway, I'm going to move some things around today and this will likely end up in its own file or in a new module called types (or something better?).